### PR TITLE
Fixed page structure | Add Download Button

### LINKED ISSUE #130: Add download buttons to Convince My Boss
@rajataaron suggested we should add download buttons for Convince my Boss page where users could easily download the text in .docx or plain-text format. 

Hence this has been implemented in the PR #119

### PR DESCRIPTION
**Pull request by vipulgupta2048**
- Fixes #130 
- Fix page structure 
- Add files for Convince my Boss
- Add missing code snippet that got lost because of conflict

@thisisayush Thanks for the help !!

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@ananyo2012 Please check anything further that we need to change in the page.  
Preview - https://vipulgupta2048.github.io/inpycon2018/convince-my-boss.html

File: convince-my-boss.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+* It's a unique chance to create strategic partnerships and promote our company's brand at the forefront of the Python Community in India, by gathering valuable insights on current market trends and also recruiting fresh talent.
+* PyCon India is also a learning opportunity, offering a wide array of talks, workshops, keynotes, developer sprints strung together in a packed schedule each day divided by various tracks that people could follow.
+* Each presented by industry experts, community leaders and many young minds delving into new technologies and latest trends in the market. Some including ML/AI, Automation, Embedded Systems, Networking, Security, Web Development.
+* I will also be meeting top technical minds, brainstorm with them on ideas, challenges, and best practices. Collaborate with them on projects and solve technical issues.
+It is THE conference to go to for Python professionals. You can learn more about the conference and last year's talk and sessions here.
```
Comment (ananyo2012) [inline convince-my-boss.txt]: Link in `here` missing. Use a link in brackets

File: convince-my-boss.html [hunk 2]
```diff
@@ -126,22 +138,31 @@ <h2 class="small-caps center">Convince My Boss</h2>
...
+      </div>
+      <div id="download" class="d-block text-center">
+        <a href="convince-my-boss.docx" class="button btn-download" download="convince-my-boss.docx" id="magic" onclick="happy()">
+          <i class="fa fa-cloud-download icon"></i>Download Document</a>
+        <a href="convince-my-boss.txt" class="button btn-download" download="convince-my-boss.txt" id="magic1" onclick="happy1()">
+          <i class="fa fa-file icon"></i>Download Plain-text</a>
+
+        <script>
+          function happy() {
+            document.getElementById("magic").innerHTML = "Happy Convincing !!";
```
Comment (ananyo2012) [inline convince-my-boss.html]: Uh the button actually looks better without this effect.

**Comment by realslimshanky** [comment]
@vipulgupta2048 any update on this PR?

**Comment by vipulgupta2048** [comment]
I just saw this, fixing. 
Sorry for the delay @ananyo2012

File: convince-my-boss.html (hunk 2 above)
Comment (vipulgupta2048) [reply convince-my-boss.html]: Removing then..

File: convince-my-boss.txt (hunk 1 above)
Comment (vipulgupta2048) [reply convince-my-boss.txt]: Good Catch.

**Comment by vipulgupta2048** [comment]
@ananyo2012 Ready to go.

**Comment by ananyo2012** [comment]
@vipulgupta2048 Seems like you didn't put the button css changes in the PR while they are there in your [gh-pages](https://github.com/vipulgupta2048/inpycon2018/blob/gh-pages/custom/css/buttons.css#L1518-L1533) branch. Please make a hotfix for this asap. The buttons don't look good without the css. The change is already deployed live in the website.

**Pull request merged**